ketchup/main.py

```diff
@@ -205,11 +195,7 @@ def run():
             print(f"Alternatively, you can watch {episodes_per_day_alternative}"
                   f" episodes a day, which is "
                   f"{episodes_per_day_alternative * per_week_limit} episodes "
-                  f"per week. So you could finish within approximately "
-                  f"{episodes_per_day_alternative} weeks.\n")
-
-            print(f"Watching {episodes_per_day_alternative * per_week_limit} "
-                  f"episodes per week instead, you can finish by "
+                  f"per week. So you could finish by "
```

Comment:
Simplified the print output to contain less information. Logically it didn't make sense. Functionality needs to be implemented to allow the user to determine how many episodes per day they wish to watch, as well as how many days out of the week (Monday - Sunday) they would like to watch also, and for the function to retain the information of how many episodes per day they wish to watch.